pkg/sfu/utils/rangemap.go

```diff
@@ -40,12 +42,23 @@ type valueType interface {
 	uint32 | uint64
 }
 
+// ---------------------------------------------------
+
 type rangeVal[RT rangeType, VT valueType] struct {
 	start RT
 	end   RT
 	value VT
 }
 
+func (r rangeVal[RT, VT]) MarshalLogObject(e zapcore.ObjectEncoder) error {
+	e.AddUint64("start", uint64(r.start))
+	e.AddUint64("end", uint64(r.end))
+	e.AddUint64("value", uint64(r.value))
```

Comment:
Could not figure out an elegant way to do this for generic types. So, casting it to the biggest size.